code/modules/client/preference/character.dm

```diff
@@ -454,6 +464,8 @@
 	h_grad_colour = query.item[53]
 	h_grad_alpha = query.item[54]
 	var/custom_emotes_tmp = query.item[55]
+	physique = query.item[56]
+	height = query.item[57]
```

Comment:
This wont work.

![image](https://github.com/ParadiseSS13/Paradise/assets/25063394/06685f91-dd61-40ed-b96c-dd8706ced1c5)

You select the columns halfway in the query. You must either:
- Move those selects to the end of the query
- Re-offset things in here

Reply:
should be handled